### PR TITLE
adding op-conductor documentation

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

- Adding op-conductor configuration, rpc, and start up documentation
- Please review against: https://www.notion.so/oplabs/op-conductor-Mainnet-Launch-Runbook-2948963ba556449e8c80d81d5b895b36

### CODE REVIEW

**Review by brokewhale — APPROVED** [review]
It looks good to me. I stumbled upon this while doing some reading on OP-Conductor.

**Comment by sbvegan** [comment]
Thank you @brokewhale! I'm going to wait for someone on our devinfra team to review this as well before we merge it in

File: pages/builders/chain-operators/tools/op-conductor.mdx [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+  | Sequencer   | OP\_CONDUCTOR\_PAUSED | OP\_NODE\_SEQUENCER\_ENABLED | OP\_NODE\_CONDUCTOR\_ENABLED |
+  | ----------- | --------------------- | ---------------------------- | ---------------------------- |
+  | sequencer-0 | true                  | true                         | false                        |
+  | sequencer-1 | true                  | false                        | false                        |
+  | sequencer-2 | true                  | false                        | false                        |
+
+  {<h3>Enable conductor on sequencers</h3>}
+
+  *   Set `OP_NODE_CONDUCTOR_ENABLED=true` on the sequencers' `op-node` instances
+  *   Ensure configuration persistance is turned off.
```
Comment (kevin-satsuma) [inline pages/builders/chain-operators/tools/op-conductor.mdx:106]: do you mind elaborating on what `configuration persistence` means in this case?

nit: typo on persistence

File: pages/builders/chain-operators/tools/op-conductor.mdx [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+
+  {<h3>Switch the active sequencer and enable conductor</h3>}
+
+  *   Rollout `sequencer-2` as the active sequencer. You can use a blue/green
+      style deployment to switch from `sequencer-0` to activate `sequencer-2` without
+      any downtime.
```
Comment (kevin-satsuma) [inline pages/builders/chain-operators/tools/op-conductor.mdx:112]: would be helpful to link out to details on the blue/green deployment.  i'm assuming this is referencing the `admin_stopSequencer` + `admin_startSequencer` combo.

File: pages/builders/chain-operators/tools/op-conductor.mdx [hunk 3]
```diff
@@ -0,0 +1,728 @@
...
+  | Sequencer   | Conductor Role | OP\_CONDUCTOR\_PAUSED | OP\_NODE\_SEQUENCER\_ENABLED | OP\_NODE\_CONDUCTOR\_ENABLED |
+  | ----------- | -------------- | --------------------- | ---------------------------- | ---------------------------- |
+  | sequencer-0 | follower       | **false**             | false                        | true                         |
+  | sequencer-1 | follower       | **false**             | false                        | true                         |
+  | sequencer-2 | leader         | **false**             | true                         | true                         |
+
+  {<h3>Set sequencer-0 to leader</h3>}
+
+  *   Set sequencer-0 to be the leader
+  *   Confirm the state
```
Comment (kevin-satsuma) [inline pages/builders/chain-operators/tools/op-conductor.mdx:151]: how does this happen?  is it automatic after conductor is unpaused?  i thought the enabled sequencer would be the initial leader of the cluster.

**Review by kevin-satsuma — COMMENTED** [review]
(this is kevin from alchemy)

ty for the docs!  only some small notes around the `Spin up op-conductor` section

File: pages/builders/chain-operators/tools/op-conductor.mdx (hunk 1 above)
Comment (sbvegan) [reply pages/builders/chain-operators/tools/op-conductor.mdx:106]: This is a flag on the `op-node`, its called something like: `--op-node-rpc-admin-state`. This needs to be disabled. If its on, conductor will fail to boot. We'll clear this up.

File: pages/builders/chain-operators/tools/op-conductor.mdx (hunk 2 above)
Comment (sbvegan) [reply pages/builders/chain-operators/tools/op-conductor.mdx:112]: yeah its something like this, I can add some high level documentation on this process to reference.

**Review by sbvegan — COMMENTED** [review]
We're going to have the devinfra team take the lead on writing the "Spin up op-conductor" section

**Pull request merged**